draft-ietf-masque-connect-ip.md

```diff
@@ -1438,6 +1438,10 @@ There are initially two entries in this registry:
 |      ip      |  IP Proxying | This Document |
 {: #iana-suffixes-table title="New MASQUE URI Suffixes"}
 
+Designated experts for this registry are advised that they should approve all
```

Comment:
I don't think it's the expert's responsibility (or within their purview) to pick whether something can be proxied or not. The only reason the registry exists is to avoid conflicts, so in my mind the goal of the review is solely to prevent future conflicts. Are you seeing this differently?

Reply:
That makes sense. Can you propose text?